### PR TITLE
fix(client): accessibility icon component to use closed captions icon

### LINKED ISSUE #52771: Purpose of accessibility button on English video player
### Describe the Issue

There is an "accessibility" button in the bottom right corner of the video player for the new English courses. Currently, the only function of this button seems to be toggling the captions on and off. Is this button going to do anything else? If not, then I suggest we use a closed caption icon instead and change the name of the button to "closed captions" and use the `aria-pressed` attribute to manage its state.

### Affected Page

https://www.freecodecamp.org/learn/a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/task-1

### Steps to Reproduce

N/A

### Expected behavior

Captions off:
```
<button aria-label="closed captions" aria-pressed="false"> ... </button>
```
Captions on:
```
<button aria-label="closed captions" aria-pressed="true"> ... </button>
```

### Screenshots

_No response_

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Sembauke**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

| On | Off |
|--------|--------|
|image  | image |

I wonder if we should store this option in local storage? So you do not need to enable it every time?

Seems like some of the other logic can be simplified as well.

Closes #52771

### CODE REVIEW

**Review by huyenltnguyen — COMMENTED** [review]
We talked about making the pressed/unpressed state more distinguishable in the issue thread, perhaps with an underline. I'm not sure if we want to address that too or leave it for a future iteration.

File: client/src/templates/Challenges/components/scene/scene.tsx [hunk 1]
```diff
@@ -197,10 +197,11 @@ export function Scene({ scene }: { scene: FullScene }): JSX.Element {
                 {!alwaysShowDialogue && (
                   <button
                     className='scene-start-btn scene-a11y-btn'
-                    aria-label='Accessibility On/Off'
+                    aria-label='closed captions'
```
[the remark this answers is not in the document]
Comment (Sembauke) [reply client/src/templates/Challenges/components/scene/scene.tsx:200]: It wasn't localised before, but sure we can.

[the remark this answers is not in the document]
Comment (Sembauke) [reply client/src/templates/Challenges/components/scene/scene.tsx:200]: It seems to me that it is a very universal sentence, closed captions.

**Review by huyenltnguyen — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM @Sembauke 🎉 

> I wonder if we should store this option in local storage? So you do not need to enable it every time?

That's a tough call. The intent is to complete the challenges by listening without seeing the words. So I worry that people who don't **need** the CC will just enable it for all - but for the people that do need it, it will be a pain to enable it each time. Maybe something for a new issue.

**Pull request merged**